buildSrc/src/main/kotlin/pro.streem.java-conventions.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    `java-library`
+}
+
+group = "pro.streem"
+version = "2.0.1-SNAPSHOT"
```

Comment:
Shouldn't this be just `2.0.1`?

Reply:
Normally the version is "SNAPSHOT" as development happens. With every merge to `master`, a new snapshot build gets automatically published to the Sonatype Snapshot repository. When we're ready to do a release, we change the version from `2.0.1-SNAPSHOT` to `2.0.1` and tag it in git. Then that gets automatically published as a release version to Maven Central. Afterwards, we change the version from `2.0.1` to `2.0.2-SNAPSHOT` so that future merges to `master` will again cause snapshot builds to be published.

I'm sure there's a way to automate this so that the publishing process just automatically appends the `-SNAPSHOT` to the version when publishing from `master` (vs publishing from a git tag). But that's not how our GHA workflows are setup currently. So I was going for lowest effort, which was to copy the existing workflows from our other projects.

Reply:
Makes sense now